docs/design/chart_format.md

```diff
@@ -447,41 +345,8 @@ Unprefixed relative paths are not valid. For example, `foo/` is not allowed as a
-Personas:
-
-
-- Operator: Responsible for running an application in production.
-- Service Dev: Responsible for developing new definitions
-- App Dev: Developer who creates applications that make use of existing definitions, but does not create definitions.
-
-
-Stories:
-
-
-- As an operator, I want a build that is 100% reproducible (exact versions)
-- As an app dev, I want to be able to search for chart definitions using keys definied in the [chart file](#the-chart-file)...
-        * by keyword, where one app may have multiple keywords (e.g. Redis has storage, message queue)
-        * by name (meaning name of the chart), where name may be "fuzzy".
-        * by author
-        * by last updated date
-- As a service dev, I want a well-defined set of practices to follow
-- As a service dev, I want to be able to work with a team on the same definition
-- As a service dev, I want to be able to indicate when a particular definition is stable, and how stable it is
-- As a service dev, I want to indicate the role I played in building a definition
-- As a service dev, I want to be able to use all of the low-level Kubernetes kinds
-- As an operator, I want to be able to determine how stable a package is
-- As an operator, I want to be able to determine what version of Kubernetes I need to run a definition
-- As an operator, I want to determine whether a definition requires extension kinds (e.g. DaemonSet or something custom), and determine this _before_ I try to install
-- As a service dev, I want to be able to express that my definition depends on others, even to the extent that I specify the version or version range of the other definition upon which I depend
-- As a service dev, I do not want to install additional tooling to write, test, or locally run a definition (this relates to the file format in that the format should not require additional tooling)
-- As a service dev, I want to be able to store auxiliary files of arbitrary type inside of a definition (e.g. a PDF with documentation, or a logo for my product)
-- As a service dev, I want to be able to store my definition in one repository, but reference a definition in another repository
-- As a service dev, I want to embed my template inside of the code that it references. For example, I want to have the code to build a docker image and the definition to all live in the same source code repository.
-
-
-## Appendix B: The Standard Chart Repository
-We have talked about adding a standard (default) chart repository that we are the gatekeepers on. This would hold only curated/vetted charts over which we have control. The following table illustrates how we’re thinking about implementing this:
+## Appendix A: The Standard Chart Repository
+The standard (default) chart repository is maintained by the Helm team at gs://kubernetes-charts/. This holds only curated/vetted charts over which we have control. The following table illustrates how we’re thinking about implementing this:
```

Comment:
@sparkprime Just noticed that this appendix is redundant with [team-workflows.md](https://github.com/kubernetes/helm/blob/master/docs/workflow/team-workflows.md). We should remove it from this document and update the team-workflows document to refer to the maintainers of `kubernetes/charts` per the comment above.